src/components/AccountForm.vue

```diff
@@ -332,6 +332,11 @@ export default {
 					...this.autoConfig,
 				})
 			} else {
+				// Removing additional whitespaces from manual configuration hosts
+				// In order to avoid issues when copy pasting imap & smtp hosts from providers documentations,
+				// which may have whitespaces.
+				this.manualConfig.imapHost = this.manualConfig.imapHost.trim()
+				this.manualConfig.smtpHost = this.manualConfig.smtpHost.trim()
 				return this.save({
 					autoDetect: false,
 					...this.manualConfig,
```

Comment:
```suggestion
				return this.save({
					autoDetect: false,
					...this.manualConfig,
					imapHost: this.manualConfig.imapHost.trim(),
					smtpHost: this.manualConfig.smtpHost.trim(),
```

same result but without the side effects

Reply:
I didn't know that syntax was possible. Added to the changes.